draft-sharabayko-mops-srt.md

```diff
@@ -2007,6 +2013,13 @@ longer, potentially undefined time, and cause even worse tearing
 for the player. Setting higher SRT latency will help much more in the
 case when TLPKTDROP causes packet drops too often.
 
+When a receiver encounters the situation where the next packet to be played was not
+successfully received from the sender, the receiver will "skip" the packet
+and send a fake ACK. To the sender, this fake ACK is a real ACK({{packet-acks}}), 
```

Comment:
I am not 100% sure, but likely space before the brace would make sense, as the reference renders into "Secition #.#".
Please check. One more such case is above.

```suggestion
and send a fake ACK. To the sender, this fake ACK is a real ACK ({{packet-acks}}), 
```